### PR TITLE
Implemented redirection for signed in users for verb-widget. Refactor

### PR DESCRIPTION
**Pull request by saragajic11**
Resolves: [MWPW-154530](https://jira.corp.adobe.com/browse/MWPW-154530)

### CODE REVIEW

File: acrobat/blocks/verb-widget/verb-widget.js [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+  sendforsignature: 'sendforsignature',
+  'rotate-pages': 'rotatepages',
+  fillsign: 'fillsign',
+  'split-pdf': 'split',
+  'insert-pdf': 'insert',
+  'compress-pdf': 'compress',
+  'png-to-pdf': 'jpgtopdf',
+  'number-pages': 'number',
+  'ocr-pdf': 'ocr',
+  'chat-pdf': 'chat',
```
Comment (TsayAdobe) [inline acrobat/blocks/verb-widget/verb-widget.js:30]: How about study-with-acrobat?

**Review by TsayAdobe — APPROVED** [review]

File: acrobat/blocks/verb-widget/verb-widget.js [hunk 2]
```diff
@@ -46,6 +71,23 @@ const setDraggingClass = (widget, shouldToggle) => {
   shouldToggle ? widget.classList.add('dragging') : widget.classList.remove('dragging');
 };
 
+function redDir(verb) {
+  if (isOldBrowser()) {
```
Comment (Blainegunn) [inline acrobat/blocks/verb-widget/verb-widget.js:75]: let's find a away to call `isOldBrowser()` regardless if the user is signed-in please.

Comment (saragajic11) [reply acrobat/blocks/verb-widget/verb-widget.js]: Hi @Blainegunn I've added it at the top of init function, please have a look and let me know what do you think about it
https://github.com/adobecom/dc/pull/800/commits/8e4d5ed908605f204ad4b08efba1aa9a961a1af4

File: acrobat/blocks/verb-widget/verb-widget.js (hunk 1 above)
Comment (saragajic11) [reply acrobat/blocks/verb-widget/verb-widget.js:30]: Thanks @TsayAdobe , i"ve added it: https://github.com/adobecom/dc/pull/800/commits/ed5d2b4fc656e781491597d74ba819a81c565dd4

**Review by Blainegunn — APPROVED** [review]

**Pull request merged**